apps/AEPSampleAppNewArchEnabled/android/app/src/main/java/com/AEPSampleAppNewArchEnabled/MainActivity.kt

```diff
@@ -54,17 +54,18 @@ class MainActivity : ReactActivity() {
           ){})
   }
 
-    // to do - add lifecycle methods
-//     override fun onResume() {
-//        super.onResume()
-//        MobileCore.setApplication(application);
-//        MobileCore.lifecycleStart(null);
-//    }
+    // Called when the main activity start interacting
+    override fun onResume() {
+        super.onResume()
+        (application as MainApplication).onResume()
```

Comment:
https://developer.adobe.com/client-sdks/home/base/mobile-core/lifecycle/android/#implementing-global-lifecycle-callbacks
Can we follow the lifecycle implementation outlined in the above public documentation?

Reply:
have updated the code accordingly @yangyansong-adbe